xenopsd.conf

```diff
@@ -99,5 +99,10 @@ disable-logging-for=http
 # Workaround for ca-140252: evtchn misalignment workaround for legacy PV tools
 # ca-140252-workaround=false
 
-# Time we allow the windows guests to do additional memory ballooning before live migration
+# Xenopsd already waits for the ballooning to end but sometimes the drivers keep
+# doing it even after it believes that the ballooning should have ended.
```

Comment:
Some confusion about what "it" refers to: xenopsd or the driver: "after xenopsd believes .."